.idea/vcs.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$" vcs="hg4idea" />
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
   </component>
-</project>
+</project>
```

Comment:
Minor: can you restore the newline that was removed? It isn't related to the change, and in general, text files should end with a newline (although that's a suggestion, not a rule).

Reply:
Updated.